README.md

```diff
@@ -87,8 +88,9 @@ X = pd.read_csv('data/MNIST-1k.csv', sep=' ', header=None).to_numpy()
 X_tsne = TSNE(n_components = 2).fit_transform(X)
 
 # Fit the data with BanditPAM:
-kmed = KMedoids(n_medoids = 10, algorithm = "BanditPAM")
-kmed.fit(X, 'L2', 10, "mnist_log")
+k = 10
+kmed = KMedoids(n_medoids = k, algorithm = "BanditPAM")
+kmed.fit(X, 'L2', k, "mnist_log")
```

Comment:
Same here; passing the argument `k` is redundant because it is already passed in Line 92